### ISSUE TITLE
`d2sitems`: parse class specific data

### ISSUE THREAD

**Issue by gucio321** [gucio321/d2d2s#36]
there is `Item.ClassSpecific` substructure. We need to parse Data to something human-readable

**Comment by Eneitilyn** [comment]
Hello I would like to make a suggestion [Kaitai-Struct](http://doc.kaitai.io/) This tool is very useful when trying to 'define'/parse/encode/decode/'make human readable sense' of files's structures according to Masumi Nakamura's guide on this tool(link found the website) by the way this also it could possibly save u a lot of time if u use it properly I think therefore I suggest maybe try it out? There is an actual Documentaiton! Also here is an example .ksy made by me for .d2s its probably no shape or form correct but u get the jist [d2s.zip](https://github.com/gucio321/d2d2s/files/7476671/d2s.zip)  its like a ini configuration file... You can also try it out in kaitai's WebIDE if u don't  want to go into the trouble of trying to set it up in ur own IDE it also would be great to have one d2s parser not to parse Diablo 2's text files for item/property/quest data/etc and kind of turn the d2s into actual readable table of content without requiring game files its gonna be flexible working on almost any d2s version kind of like how pairofdocs/d2s_edit_recalc's patch list works but much bigger? I could maybe msg u somewhere else for more ideas/convenience...trying not to post big essays on github...

**Comment by gucio321** [comment]
wow @Eneitilyn it looks really nice. I'll deffinitly investigate it.
I didn't know about this solution and it may be really helpful in my work.
The reason however, why I'm not g oing to totally switch to ksy is, that it creates only data **parser**, when the idea of my project is to create fully reversible process, when the user can parse d2s file to a human-readable structure and then, write it back to d2s format.

**Comment by Eneitilyn** [comment]
Yeah sadly Kaitai doesn't seem to support Go language yet X_X which makes it kind of harsh completely swapping to ksy cuz it basically requires you to rewrite the entire project or something of the sort probably...Don't want to ask questions offtopic here by the way @gucio321  may I talk to you in Discord possibly?

**Comment by gucio321** [comment]
> Yeah sadly Kaitai doesn't seem to support Go language yet

it does

>  @gucio321 may I talk to you in Discord possibly?

sure! I'm `M.Sz#4522`
